articles/azure/azs-how-resize-unmanaged-disk.md

```diff
@@ -35,14 +35,14 @@ From your PowerShell window:
 > [!IMPORTANT]
 > Variables to be changed:
```

Comment:
Change to:
Enter details below to provide values for the variables in the scripts in this article

Reply:
Fixed